lib/doorkeeper-jwt.rb

```diff
@@ -40,8 +40,28 @@ def self.rsa_encryption?
       /RS\d{3}/ =~ encryption_method
     end
 
+    def self.ecdsa_encryption?
```

Comment:
`private` (on line 15) does not make singleton methods private. Use `private_class_method` or `private` inside a `class << self` block instead.